src/browser/animations.js

```diff
@@ -216,16 +223,13 @@ function AnimationHandler(desiredInterval) {
 
                     if (updateBounds) {
                         currentWindowEntry.hadBoundsChange = true;
-                        var newBounds;
-                        /* jshint bitwise:false */
-                        var initialBounds = currentTransition.initialBounds;
-                        newBounds = clipBounds({
-                            x: (initialBounds.x + boundsChange.x) | 0,
-                            y: (initialBounds.y + boundsChange.y) | 0,
-                            width: (initialBounds.width + boundsChange.width) | 0,
-                            height: (initialBounds.height + boundsChange.height) | 0
-                        }, bw);
-                        /* jshint bitwise:true */
+
+                        let { x, y, width, height } = currentTransition.initialBounds;
+                        x = toSafeInt(x + boundsChange.x, x);
```

Comment:
purposely did not add a new line because in this case all 3 lines together are making up the definition of x and y